### ISSUE TITLE
Reverse level

### ISSUE THREAD

**Issue by felixsanz** [valeriangalliat/markdown-it-anchor#27]
Doesn't `level` works as opposite as its supposed to?

For example, i want to give an id attribute to my main headers, h1, h2 and h3, but i can't use the `level` attribute to control that. How i'm supposed to exclude the h4 ones? It should work the opposite :/

`level`: 3 <- minimum header level of 3 to apply the functionality.

But right now that's just adding the attributes to h3-h6 tags, which is... weird.

**Comment by valeriangalliat** [comment]
Makes sense. Initially my thought was mostly to exclude `h1` because the page I was working on had only one `h1` and I didn't want give it an anchor, that's why it works this way.

But it think it would be a good idea to make `level` more flexible, supporting something like `level: [1, 3]` in your case.

I'll give it a shot over the week-end.

**Comment by felixsanz** [comment]
Thats great, an array with the levels where you want to have ids/anchors.

My issue is because i have headings like:

```html
<h4><code>options</code></h4>
<ul>
<li>active: true</li>
<li>...</li>
</ul>
```

And because my heading doesn't contain text, it's generated like `<h4 id=""><code>...</code></h4>`, which is invalid. So i want to disable this functionality in <h4> headings, where i usually put code tags.

**Comment by valeriangalliat** [comment]
Oh, interesting, I'd also consider this as a bug, the ID should be a slugified version of the code too. I'm looking into this.

**Comment by valeriangalliat** [comment]
Hmm, I just tested the following:

```
equal(
  md({ html: true }).use(anchor).render('#### <code>options</code>'),
  '<h4 id="options"><code>options</code></h4>\n'
)
```

And it actually passes. What versions of markdown-it and markdown-it-anchor are you using?

**Comment by valeriangalliat** [comment]
I fixed both issues; the inline code is now also used to generate the ID, and you can pass an array of header levels to select in the `level` option.

It's published as of 4.0.0.

Note that I also dropped Babel support in 4.0.0 so this module only works with a Node version with ES6 support.

**Comment by felixsanz** [comment]
Works great, thanks!
